plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -183,10 +188,12 @@ def _validate_multi_sig(self, multi_sig: MultiSignature):
                                                                   public_keys)
 
     def _sign_state(self, pre_prepare: PrePrepare):
-        pool_root_hash_ser = self.state_root_serializer.serialize(
-            bytes(self._bls_bft.bls_key_register.get_pool_root_hash_committed()))
+        pool_root_hash = pre_prepare.poolStateRootHash \
```

Comment:
I think it would be better to create a helper method (in this class) to get pool_root_hash with PrePrepare as an input

Reply:
Added BlsBftReplicaPlenum._get_pool_root_hash()